deployments/pod-sriovdp.yaml

```diff
@@ -19,16 +21,29 @@ spec:
     - mountPath: /etc/pcidp/
       name: config
       readOnly: true
+    - name: hwdata
+      mountPath: /usr/share/hwdata
```

Comment:
you are assume the lspci is installed? it is install in all OS on the same location?

Reply:
> you are assume the lspci is installed? it is install in all OS on the same location?

The 'ghw' library uses /usr/share/hwdata/pci.ids file. This file is part of hwdata package available in rpm and deb. As a fallback the library will try to download this database from https://pci-ids.ucw.cz/v2.2/pci.ids.gz. 
We may as well add the 'hwdata' as a prerequisites on the host.

Reply:
in my ubuntu is it under /usr/share/misc/pci.ids

Reply:
Which release?
On 17.04 I can see there's a symlink
```
root@silv-ah-ubuntu1704:~# ll /usr/share/hwdata/
total 72
drwxr-xr-x   2 root root  4096 Mar 25 11:49 ./
drwxr-xr-x 306 root root 12288 Mar 25 11:49 ../
lrwxrwxrwx   1 root root    15 Jul  3  2016 pci.ids -> ../misc/pci.ids
-rw-r--r--   1 root root 55055 Jul  3  2016 pnp.ids
lrwxrwxrwx   1 root root    15 Jul  3  2016 usb.ids -> ../misc/usb.ids
```

Reply:
Ubuntu 18.04.2 LTS